### PR TITLE
Added code for class and constructors

### PR DESCRIPTION
**Pull request by deepakghadge19**
Added code for classes and constructors

### CODE REVIEW

File: src/class_Constructor_AccessModifiers_17March2023/CuboidConstructorDemo.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package class_Constructor_AccessModifiers_17March2023;
+
+public class CuboidConstructorDemo {
+	int width;
```
Comment (vrushalee123) [inline src/class_Constructor_AccessModifiers_17March2023/CuboidConstructorDemo.java:4]: declare variables as private

File: src/class_Constructor_AccessModifiers_17March2023/CuboidConstructorDemo.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	int height;
+	int depth;
+
+	CuboidConstructorDemo(int width, int height, int depth) {
```
Comment (vrushalee123) [inline src/class_Constructor_AccessModifiers_17March2023/CuboidConstructorDemo.java:8]: Make constructor public

File: src/class_Constructor_AccessModifiers_17March2023/StudentClassDemo.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
+package class_Constructor_AccessModifiers_17March2023;
+
+public class StudentClassDemo {
+
+	String name;
```
Comment (vrushalee123) [inline src/class_Constructor_AccessModifiers_17March2023/StudentClassDemo.java:5]: Make variable always as private.

**Review by vrushalee123 — CHANGES_REQUESTED** [review]

File: src/class_Constructor_AccessModifiers_17March2023/CuboidConstructorDemo.java (hunk 1 above)
Comment (deepakghadge19) [reply src/class_Constructor_AccessModifiers_17March2023/CuboidConstructorDemo.java]: Done! Declared variable as private

File: src/class_Constructor_AccessModifiers_17March2023/CuboidConstructorDemo.java (hunk 2 above)
Comment (deepakghadge19) [reply src/class_Constructor_AccessModifiers_17March2023/CuboidConstructorDemo.java]: Done

**Review by amitsahu8 — APPROVED** [review]

**Pull request merged**